src/services/gui/gui_service.hpp

```diff
@@ -190,4 +190,4 @@ namespace big
 	};
 
 	inline gui_service* g_gui_service{};
-}
\ No newline at end of file
+}
```

Comment:
Fix nonsense whitespace changes. Don't you guys check these changes before pushing?

Reply:
this last one i did not add. i just made the one change on github page itself. I believe github added it.

Reply:
nvm its github bug. i locally pushed the changes now